src/main/java/io/vertx/proton/impl/ProtonTransport.java

```diff
@@ -100,18 +100,18 @@ private void handleSocketBuffer(Buffer buff) {
 
       switch (eventType) {
       case CONNECTION_REMOTE_OPEN: {
-        connnection.fireRemoteOpen();
+        connection.fireRemoteOpen();
```

Comment:
It was obviously incorrect before, but it makes things easier to review if unrelated things are fixed separately.